cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -381,21 +377,15 @@ impl NetworkRunnable for Cmd {
         let (return_value, events) = if self.is_view() {
             (sim_res.results()?[0].xdr.clone(), sim_res.events()?)
         } else {
-            let global::Args {
-                verbose,
-                very_verbose,
-                no_cache,
-                ..
-            } = global_args.cloned().unwrap_or_default();
+            let global::Args { no_cache, .. } = global_args.cloned().unwrap_or_default();
+            // Need to sign all auth entries
+            let mut txn = txn.transaction().clone();
+            if let Some(tx) = config.sign_soroban_authorizations(&txn, &signers).await? {
+                txn = tx;
+            }
+            // let txn = self.config.sign_with_local_key(tx)
             let res = client
-                .send_assembled_transaction(
-                    txn,
-                    &unwrap_config.key_pair()?,
-                    &signers,
-                    &network.network_passphrase,
-                    Some(log_events),
-                    (verbose || very_verbose || self.fee.cost).then_some(log_resources),
```

Comment:
Are we no longer logging events and resources?

Reply:
It was removed from the RPC crate. My thoughts here was that we would move the logging here since the RPC crate shouldn't do any logging imo. I'll add back the event logging. However, the resources should be now part of the action cache.

Reply:
Can we keep the logs for both the events and resources?

Even though resources are in the action cache, folks could still be invoking with logs enabled to visually see resources.